consensus/expected.go

```diff
@@ -265,18 +267,12 @@ func (c *Expected) RunStateTransition(ctx context.Context, ts types.TipSet, ance
 // validateMining checks validity of the block ticket, proof, and miner address.
 //    Returns an error if:
 //    	* any tipset's block was mined by an invalid miner address.
-//      * the block ticket is incorrectly computed
+//      * the block proof is invalid for the challenge
 //      * the block ticket fails the power check, i.e. is not a winning ticket
 //    Returns nil if all the above checks pass.
 // See https://github.com/filecoin-project/specs/blob/master/mining.md#chain-validation
 func (c *Expected) validateMining(ctx context.Context, st state.Tree, ts types.TipSet, parentTs types.TipSet) error {
 	for _, blk := range ts.ToSlice() {
-		computedTicket := CreateTicket(blk.Proof, blk.Miner)
```

Comment:
validating mining will no longer compute a ticket since tickets have to be signed, and it can't (and shouldn't) be signed in verification.